gateway/webhook/webhook_test.go

```diff
@@ -29,8 +29,9 @@ const (
 )
 
 var (
-	whStats *webhookStatsT
-	once    sync.Once
+	whStats           *webhookStatsT
+	once              sync.Once
+	transformerServer = httptest.NewServer(http.HandlerFunc(transformMockHandler))
```

Comment:
isn't it simpler each test to have its own server? Tests run in paraller during ci

Reply:
problem here is, serverUrl is passed using common variable: sourceTransformerURL so we can't have server per test so I have written in generic way test.

Reply:
we may refactor `sourceTransformerURL` and convert it to a `batchWebhookTransformerT` field, correct?

Reply:
+1 for removing the global variable or simply injecting it as a struct attribute at construction time.

Reply:
Done, please check if this is reasonable, for tests I had to add variadic params so that we can dynamically set sourceTransformerURL.